pyston/resource.py

```diff
@@ -202,11 +202,19 @@ def cors_max_age(self):
     def _demap_key(self, lookup_key):
         return {v: k for k, v in self.DATA_KEY_MAPPING.items()}.get(lookup_key, lookup_key)
 
-    def _flatten_dict(self, dct):
-        return {self._demap_key(str(k)): dct.get(k) for k in dct.keys()} if isinstance(dct, dict) else {}
+    def update_serialized_data(self, data):
+        if data and self.DATA_KEY_MAPPING:
+            data = LazyMappedSerializedData(data, self.DATA_KEY_MAPPING).serialize()
+        return data
+
+    def update_deserialized_data(self, data):
+        return (
+            {self._demap_key(k): v for k, v in data.items() if k not in self.DATA_KEY_MAPPING}
```

Comment:
tady byla celkem slušná chybka, když se data poslali ve snake_case tak to fungovalo